### DISCUSSION TITLE
Next release tech stack samples

### DISCUSSION THREAD

**Discussion by luisaveiro** [luisaveiro/playground#1] [Polls]
The playground repository includes 60 samples, including JavaScript runtimes, popular frameworks, headless CMS, and "Hello, World!" examples of the most common programming languages. 

Although I have my wishlist for the list of samples that I want to include, I would like feedback on the categories of tech stack samples for the next release. I want my playground repo to provide value to developers by reducing the set-up complexity that some tech stacks require to create a new project. I have experienced some frustration while setting up some of the samples. Now the set-up is easy.

I believe coding must be an enjoyable and creative experience. It would be great if my playground repo could encourage developers to learn new tech stacks.

Do you have a specific tech stack in mind? Please comment below with your suggestion and let's try having the tech stack included in this release.

**Comment by geoman2** [comment]
Hi Luis, is there a chance to integrate a hybrid mobile framework based on flutter shell with some of these web apps/frameworks? There is a tutorial https://github.com/Backendless/native-mobile-app-shell

**Reply by luisaveiro** [reply]
Hi @geoman2, thanks for sharing the repo. I have been researching how I can configure hybrid mobile frameworks to use Docker. I'm testing a few options.

**Comment by luisaveiro** [comment]
I have added Soketi (WebSockets) in [v0.12.0](https://github.com/luisaveiro/playground/releases/tag/v0.12.0).
